src/data/ProgramsD2Repository.ts

```diff
@@ -72,3 +114,9 @@ interface ApiSaveResponse {
 interface Report {
     message: string;
 }
+
+function delay(t: number) {
+    return new Promise(resolve => setTimeout(resolve, t));
+}
+
+export type TrackedEntityInstancePick = Pick<TrackedEntityInstance, "enrollments" | "trackedEntityInstance">;
```

Comment:
`TrackedEntityInstancePick`. `Pick` is somewhat nondescript. Not that I have a proposal :)